python/paddle/fluid/layers/control_flow.py

```diff
@@ -912,6 +913,71 @@ def _complete(self):
                    "is_test": self.is_test})
 
 
+def while_loop(cond, body, loop_vars, name=None):
+    """
+    This while_loop operator control. Repeat while_loop body until cond return False.
+
+    Args:
+        cond(Callable): A callable returning a boolean tensor controlling whether to continue looping.
+	body(Callable): A callable returning a tuple, namedtuple or list of tensors of the same arity(length and
+            structure) and types as `loops_vars`.
+	loop_vars(list|tuple): A list ,namedtuple or tuple of tensors that is passed to both `cond` and `body`.
+	is_test(bool, optional): A flag indicating whether execution is in test phase. Default is None.
+	name(str, optional): Normally there is no need for users to set this property. For more information, please
+            refer to :ref:`api_guide_Name`. Default is None.
+    Examples:
+        .. code-block:: python
+
+	import paddle.fluid as fluid
```

Comment:
Example indent fault. It can cause fault in python

Reply:
Done.